styx-common/src/main/java/com/spotify/styx/util/ReplayEvents.java

```diff
@@ -59,7 +59,7 @@ private ReplayEvents() {
       final long lastConsumedEvent = entry.getValue();
       final SettableTime time = new SettableTime();
       if (printLogs) {
-        LOG.info("Replaying {} up to #{}", workflowInstance.toKey(), lastConsumedEvent);
+        LOG.debug("Replaying {} up to #{}", workflowInstance.toKey(), lastConsumedEvent);
```

Comment:
Not sure. Need to look at the call sites and understand why it `printLogs` was introduced in the first place I guess.